packages/sdk/contractkit/src/wrappers/Escrow.test.ts

```diff
@@ -0,0 +1,105 @@
+import { testWithGanache } from '@celo/dev-utils/lib/ganache-test'
+import { newKitFromWeb3 } from '../kit'
+import { EscrowWrapper } from './Escrow'
+import { FederatedAttestationsWrapper } from './FederatedAttestations'
+import { StableTokenWrapper } from './StableTokenWrapper'
+import { getParsedSignatureOfAddress } from '@celo/protocol/lib/signing-utils'
+
+testWithGanache('Escrow Wrapper', (web3) => {
+  const kit = newKitFromWeb3(web3)
+  const TEN_CUSD = kit.web3.utils.toWei('10', 'ether')
+
+  let accounts: string[] = []
+  let escrow: EscrowWrapper
+  let federatedAttestations: FederatedAttestationsWrapper
+  let stableTokenContract: StableTokenWrapper
+  let identifier: string
+
+  beforeAll(async () => {
+    escrow = await kit.contracts.getEscrow()
+    stableTokenContract = await kit.contracts.getStableToken()
+    federatedAttestations = await kit.contracts.getFederatedAttestations()
+
+    accounts = await web3.eth.getAccounts()
+    kit.defaultAccount = accounts[0]
+
+    identifier = kit.web3.utils.soliditySha3({
+      t: 'bytes32',
+      v: kit.web3.eth.accounts.create().address,
+    }) as string
+  })
+
+  it('transfer with trusted issuers should set TrustedIssuersPerPayment', async () => {
+    const testPaymentId = kit.web3.eth.accounts.create().address
+    const testIssuedOnTimestamp = Math.floor(new Date().getTime() / 1000)
```

Comment:
using new Date() (or other changing timestamps) in tests leads to bugs that only happen in certain timezones or certain times of the year